SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -197,8 +198,103 @@ void CTurbSSTSolver::Preprocessing(CGeometry *geometry, CSolver **solver_contain
          unsigned short iMesh, unsigned short iRKStep, unsigned short RunTime_EqSystem, bool Output) {
   SU2_OMP_SAFE_GLOBAL_ACCESS(config->SetGlobalParam(config->GetKind_Solver(), RunTime_EqSystem);)
 
+  const auto kind_hybridRANSLES = config->GetKind_HybridRANSLES();
+
   /*--- Upwind second order reconstruction and gradients ---*/
   CommonPreprocessing(geometry, config, Output);
+
+  if (kind_hybridRANSLES != NO_HYBRIDRANSLES) {
+
+    /*--- Set the vortex tilting coefficient at every node if required ---*/
+
+    if (kind_hybridRANSLES == SST_EDDES){
+      auto* flowNodes = su2staticcast_p<CFlowVariable*>(solver_container[FLOW_SOL]->GetNodes());
+
+      SU2_OMP_FOR_STAT(omp_chunk_size)
+      for (unsigned long iPoint = 0; iPoint < nPoint; iPoint++){
+        auto Vorticity = flowNodes->GetVorticity(iPoint);
+        auto PrimGrad_Flow = flowNodes->GetGradient_Primitive(iPoint);
+        auto Laminar_Viscosity = flowNodes->GetLaminarViscosity(iPoint);
+        nodes->SetVortex_Tilting(iPoint, PrimGrad_Flow, Vorticity, Laminar_Viscosity);
+      }
+      END_SU2_OMP_FOR
+    }
+
+    /*--- Compute the DES length scale ---*/
+
+    SetDES_LengthScale(solver_container, geometry, config);
+
+  }
+
+  if (sstParsedOptions.sasModel == SST_OPTIONS::SAS_BABU){
+
+    auto* flowNodes = su2staticcast_p<CFlowVariable*>(solver_container[FLOW_SOL]->GetNodes());
+    
+    SU2_OMP_FOR_DYN(256)
+    for (unsigned long iPoint = 0; iPoint < nPoint; ++iPoint) {
+
+      /*--- Initialize. ---*/
+      for (unsigned short iDim = 0; iDim < nDim; iDim++)
+        nodes->SetVelLapl(iPoint, iDim, 0.0);
+
+      const su2double halfOnVol = 1.0 / (geometry->nodes->GetVolume(iPoint) + geometry->nodes->GetPeriodicVolume(iPoint));
+
+      const auto coordsIPoint = geometry->nodes->GetCoord(iPoint);
+
+      /*--- Loop over the neighbors of point i. ---*/
+      for (size_t iNeigh = 0; iNeigh < geometry->nodes->GetnPoint(iPoint); ++iNeigh) {
+
+        size_t iEdge = geometry->nodes->GetEdge(iPoint, iNeigh);
+        size_t jPoint = geometry->nodes->GetPoint(iPoint, iNeigh);
+
+        /*--- Determine if edge points inwards or outwards of iPoint.
+        *    If inwards we need to flip the area vector. ---*/
+
+        const su2double weight = halfOnVol;
+
+        const auto area = geometry->edges->GetNormal(iEdge);
+        AD::SetPreaccIn(area, nDim);
```

Comment:
This is not doing anything because you are not starting/ending the preaccumulation.